src/packages/cli/src/__tests__/integration/mssql/__scenarios.ts

```diff
@@ -0,0 +1,1559 @@
+import { Input } from '../../__helpers__/integrationTest'
+
+export const scenarios = [
+  {
+    name: 'findOne where PK',
+    up: `
+        create table teams (
+          id int primary key not null,
+          name varchar(32) not null unique
+        );
+        insert into teams (id, name) values (1, 'a');
+        insert into teams (id, name) values (2, 'b');
+      `,
+    do: async client => {
+      return client.teams.findOne({ where: { id: 2 } })
+    },
+    expect: {
+      id: 2,
+      name: 'b',
+    },
+  },
+  {
+    name: 'findOne where PK with select',
+    up: `
+        create table teams (
+          id int primary key not null,
+          name varchar(32) not null unique,
+          email varchar(32) not null unique
+        );
+        insert into teams (id, name, email) values (1, 'a', 'a@a');
+        insert into teams (id, name, email) values (2, 'b', 'b@b');
+      `,
+    do: async client => {
+      return client.teams.findOne({
+        where: { id: 2 },
+        select: { name: true },
+      })
+    },
+    expect: {
+      name: 'b',
+    },
+  },
+  {
+    name: 'findOne where PK with include',
+    up: `
+        create table users (
+          id int identity primary key not null,
+          email varchar(32) not null unique
+        );
+        create table posts (
+          id int identity primary key not null,
+          user_id int not null references users (id) on update cascade,
+          title varchar(32) not null
+        );
+        insert into users ("email") values ('ada@prisma.io');
+        insert into users ("email") values ('ema@prisma.io');
+        insert into posts ("user_id", "title") values (1, 'A');
+        insert into posts ("user_id", "title") values (1, 'B');
+        insert into posts ("user_id", "title") values (2, 'C');
+      `,
+    do: async client => {
+      return client.users.findOne({
+        where: { id: 1 },
+        include: { posts: true },
+      })
+    },
+    expect: {
+      email: 'ada@prisma.io',
+      id: 1,
+      posts: [
+        {
+          id: 1,
+          title: 'A',
+          user_id: 1,
+        },
+        {
+          id: 2,
+          title: 'B',
+          user_id: 1,
+        },
+      ],
+    },
+  },
+  {
+    name: 'create with data',
+    up: `
+        create table teams (
+          id int identity primary key not null,
+          name varchar(32) not null unique
+        );
+      `,
+    do: async client => {
+      return client.teams.create({ data: { name: 'c' } })
+    },
+    expect: {
+      id: 1,
+      name: 'c',
+    },
+  },
+  {
+    name: 'create with empty data and SQL default',
+    up: `
+        create table teams (
+          id int identity primary key not null,
+          name varchar(32) not null default 'alice'
+        );
+      `,
+    do: async client => {
+      return client.teams.create({ data: {} })
+    },
+    expect: {
+      id: 1,
+      name: 'alice',
+    },
+  },
+  {
+    todo: true,
```

Comment:
Has this also been a todo in the other dbs?

Reply:
Ha yeah, forgot to bring this up. I can send a separate PR implementing this test everywhere else too.

Reply:
We have a few todo indeed, I would consider this non blocking and an improvement